electron/ui/src/views/FlowsheetConfig/ConfigOutput/ConfigOutput.js

```diff
@@ -55,43 +59,6 @@ export default function ConfigOutput(props) {
         });
     }
 
-    const handleDownloadOutput = () => {
-        let headers = ['category','metric','units','value']
-        let values = []
-        for (let key of Object.keys(outputData.outputData.exports)) {
-            let each = outputData.outputData.exports[key]
-            if (each.is_output) {
-                values.push([each.output_category, each.name, each.display_units, each.value])
-            }
-        }
-        let body = {headers: headers, values:values}
-        downloadSingleOutput(params.id, body)
-        .then(response => response.blob())
-        .then((data)=>{
-            const href = window.URL.createObjectURL(data);
-            const link = document.createElement('a');
-            link.href = href;
-            link.setAttribute('download', `${outputData.inputData.name}_output.csv`);
-            document.body.appendChild(link);
-            link.click();
-            document.body.removeChild(link);
-        });
-    }
-
-    const downloadSweepOutput = () => {
-        downloadSweepResults(params.id)
-        .then(response => response.blob())
-        .then((data)=>{
-            const href = window.URL.createObjectURL(data);
-            const link = document.createElement('a');
-            link.href = href;
-            link.setAttribute('download', 'sweep_results.csv');
-            document.body.appendChild(link);
-            link.click();
-            document.body.removeChild(link);
-        });
-    }
-
```

Comment:
not sure if I'm missing something, are these download options being removed?

Reply:
I did somehow drop it! Thanks for catching that. It is re-inserted now, with some refactoring included to break out the different download sections into their own components.